.gitignore

```diff
@@ -23,6 +23,8 @@ vendor/assets/bower_components/
 /db/schema.rb
 /public/system/*
 /coverage/
+/features/req_cache/*
```

Comment:
note that these must not go in .gitignore as they are the cache files that create a sandbox for testing